rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/serialization/ExportNodeSourceToCatalogServlet.java

```diff
@@ -81,8 +81,7 @@ private void upload(HttpServletRequest request, HttpServletResponse response) {
     private void postRequestAndHandleResponse(HttpServletResponse response, CatalogObjectAction catalogObjectAction,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "catalogObjectAction". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)